node_operator.py

```diff
@@ -17,111 +18,161 @@
 from eth2deposit.settings import get_chain_setting, MAINNET, PRATER
 from eth2deposit.utils.validation import verify_deposit_data_json
 
-web3_eth = Web3(Web3.HTTPProvider(sys.argv[1]))
-web3_eth.isConnected()
-web3_eth.middleware_onion.inject(geth_poa_middleware, layer=0)
-account = web3_eth.eth.account.privateKeyToAccount(sys.argv[2])
-web3_eth.eth.account.enable_unaudited_hdwallet_features()
 
-KeysManager = "0xd7C8052A5db95BBA40F3AE3882E6127A068BAeF9"
-with open("contracts/keysmanager.json", 'r') as file:
-    a = file.read()
-keysmanager_contract = web3_eth.eth.contract(abi=a, address=Web3.toChecksumAddress(KeysManager))
-with open("contracts/deposit_contract.json", 'r') as file:
-    abi = file.read()
-depositcontract = web3_eth.eth.contract(abi=abi, address=Web3.toChecksumAddress("0xff50ed3d0ec03ac01d4c79aad74928bff48a7b2b"))
-mnemonic = get_mnemonic(language='english', words_path=WORD_LISTS_PATH)
+class NodeOperator:
 
-num_validator = int(sys.argv[4])
-keystore_password = sys.argv[3]
-print(keystore_password)
+    @staticmethod
+    def connect_to_eth(eth_endpoint, private_key):
+        web3_eth = Web3(Web3.HTTPProvider(eth_endpoint))
+        web3_eth.isConnected()
+        web3_eth.middleware_onion.inject(geth_poa_middleware, layer=0)
+        account = web3_eth.eth.account.privateKeyToAccount(private_key)
+        web3_eth.eth.account.enable_unaudited_hdwallet_features()
+        return web3_eth, account
 
+    @staticmethod
```

Comment:
Hmm a class with only static methods actually does not make sense. Why was the std functions moved to static functions here?

Reply:
Created a class so that have to initialize everything once but realised all methods have separate inputs and are not dependent on each other, created static methods. Removed the class implementation all together